cmd/tempo-vulture/main.go

```diff
@@ -332,7 +332,7 @@ func searchTag(client *util.Client, seed time.Time) (traceMetrics, error) {
 		zap.String("hexID", hexID),
 		zap.Duration("ago", time.Since(seed)),
 		zap.String("key", attr.Key),
-		zap.String("value", attr.Value.GetStringValue()),
+		zap.String("value", util.StringifyAnyValue(attr.Value)),
```

Comment:
Is there a reason why you changed this `GetStringValue` but not the one 8 lines further?

Reply:
I missed it. Added in 309793e3fe14e9817d916b0c89d46c77f40508d5